### PR TITLE
LPS-0 DO NOT MERGE Run all forms tests

### PR DESCRIPTION
**Pull request by pedroqueiroz94**

### CODE REVIEW

**Comment by pedroqueiroz94** [comment]
ci:test

File: modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance_records.jsp [hunk 1]
```diff
@@ -90,7 +90,7 @@ renderResponse.setTitle(LanguageUtil.get(request, "form-entries"));
 
 					<liferay-ui:search-container-column-text
 						name="<%= ddmFormViewFormInstanceRecordsDisplayContext.getColumnName(ddmFormField) %>"
-						value="<%= ddmFormViewFormInstanceRecordsDisplayContext.getColumnValue(ddmFormField, ddmFormFieldValuesMap.get(ddmFormField.getName())) %>"
+						value="<%= ddmFormViewFormInstanceRecordsDisplayContext.getColumnValue(formInstanceRecord, ddmFormField, ddmFormFieldValuesMap.get(ddmFormField.getName())) %>"
```
Comment (pedroqueiroz94) [inline modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance_records.jsp]: hey @linolaoj , can you please use ddmFormValues to retrieve ddmForm instead? Thanks!

**Comment by pedroqueiroz94** [comment]
ci:test

**Comment by rafaprax** [comment]
Already merged on master

**Pull request closed without merging**